From: Tiezhu Yang <yangtiezhu@loongson.cn>
Subject: include/linux/entry-common.h: remove has_signal comment of arch_do_signal_or_restart() prototype
Date: Fri, 2 Sep 2022 11:41:46 +0800

The argument has_signal of arch_do_signal_or_restart() has been removed in
commit 8ba62d37949e ("task_work: Call tracehook_notify_signal from
get_signal on all architectures"), let us remove the related comment.

Link: https://lkml.kernel.org/r/1662090106-5545-1-git-send-email-yangtiezhu@loongson.cn
Fixes: 8ba62d37949e ("task_work: Call tracehook_notify_signal from get_signal on all architectures")
Signed-off-by: Tiezhu Yang <yangtiezhu@loongson.cn>
Reviewed-by: Kees Cook <keescook@chromium.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 include/linux/entry-common.h |    1 -
 1 file changed, 1 deletion(-)

--- a/include/linux/entry-common.h~entry-remove-has_signal-comment-of-arch_do_signal_or_restart-prototype
+++ a/include/linux/entry-common.h
@@ -253,7 +253,6 @@ static __always_inline void arch_exit_to
 /**
  * arch_do_signal_or_restart -  Architecture specific signal delivery function
  * @regs:	Pointer to currents pt_regs
- * @has_signal:	actual signal to handle
  *
  * Invoked from exit_to_user_mode_loop().
  */
_
